src/main/java/com/iexec/commons/containers/client/DockerClientInstance.java

```diff
@@ -801,6 +802,56 @@ public synchronized boolean removeContainer(String containerName) {
             return false;
         }
     }
+
+    /**
+     * Retrieves the execution duration of a container.
+     * If the container has not been started yet or has not ended yet,
+     * then return {@link Optional#empty()}.
+     * Otherwise, return the duration.
+     * <p>
+     * /!\ Docker inspection command precision could lead to sub-zero execution duration for fast containers.
+     * In this case, this method return a zero-duration object.
+     *
+     * @param containerName Name of the container to look for execution duration.
+     * @return {@code Optional#empty()} if not started or ended,
```

Comment:
```suggestion
     * @return {@link Optional#empty()} if not started or ended
```

Reply:
or 
```suggestion
     * @return {@literal Optional.empty()} if not started or ended
```

Reply:
It is recommended not to repeat `@link` when already in the JavaDoc. See line 809 for. However, I would prefer to keep a similar format - `Optional#empty()` - so that it's coherent.
Regarding the coma: there's another line afterwards and I'd like to keep the separation between both with a coma.

Reply:
This is where I have a question. I understand your point and at the same time, once rendered, the `{@link ...}` becomes `Optional.empty()`. Applying my suggestion may provide a more consistent reader experience when taking this into account. We may want to take some more time to play with the different possibilities before taking the final decision

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-commons-containers/pull/29/commits/8f835e3a803f7d3b97d634ec68d85361dc336a74.
I went for a `@link` in order for everything to be coherent :)